File: app/src/main/java/com/eyalya/test/asiotech/models/Bag.java
package com.eyalya.test.asiotech.models;

public class Bag {
    public int id;
    public float weight;

    public Bag(int id, float weight) {
        this.id = id;
        this.weight = weight;
    }

    @Override
    public String toString() {
        return "Bag #" + id + " (" + weight + ")Kg";
    }
}

File: app/src/main/java/com/eyalya/test/asiotech/models/Bag.kt
package com.eyalya.test.asiotech.models

import com.eyalya.test.asiotech.common.formatToString

class Bag(var id: Int, var weight: Float) {
    override fun toString(): String {
        return "Bag #$id (${weight.formatToString()})Kg"
    }
}